src/SFA.DAS.Tasks.Worker/MessageProcessors/ApprovedTransferConnectionInvitationMessageProcessor.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Threading.Tasks;
+using MediatR;
+using SFA.DAS.EmployerAccounts.Events.Messages;
+using SFA.DAS.Messaging;
+using SFA.DAS.Messaging.AzureServiceBus.Attributes;
+using SFA.DAS.Messaging.Interfaces;
+using SFA.DAS.NLog.Logger;
+using SFA.DAS.Tasks.Application.Commands.SaveTask;
+using SFA.DAS.Tasks.API.Types.Enums;
+using SFA.DAS.Tasks.Infrastructure.Attributes;
+
+namespace SFA.DAS.Tasks.Worker.MessageProcessors
+{
+    [ServiceBusConnectionString("ManageApprenticeships")]
+    [TopicSubscription("Task_ApprovedTransferConnectionInvitationMessageProcessor")]
+
```

Comment:
topic or subscribers letter count cant be >= 50, can you change it to Task_ApprovedTransferConnectionInvitation

Reply:
Changed